lib/fetch/response.js

```diff
@@ -364,6 +364,19 @@ function makeNetworkError (reason) {
   })
 }
 
+// @see https://fetch.spec.whatwg.org/#concept-network-error
+function isNetworkError (response) {
+  return (
+    // A network error is a response whose type is "error",
+    response.type === 'error' &&
+    //  status is 0
+    response.status === 0 &&
+    'error' in response && (
+      'aborted' in response ? typeof response.aborted === 'boolean' : true
+    )
```

Comment:
what's the purpose of this? type == 'error' and a status of 0 should be enough here.